content/Konveyor/installation.md

```diff
@@ -22,43 +22,76 @@ Follow the steps below to provision minikube for single users deploying Konveyor
-
-|Name|Default Size|Access Mode|Description|
-|--|--|--|--|
-|hub database|5Gi|RWO|Hub DB|
-|hub bucket|100Gi|RWX|Hub file storage|
-|keycloak postgresql|1Gi|RWO|Keycloak backend DB|
-|pathfinder postgresql|1Gi|RWO|Pathfinder backend DB|
-|maven|100Gi|RWX|maven m2 repository|
-
-Follow the steps below to install the Konveyor Operator in the konveyor-tackle namespace (default) on any Kubernetes distribution, including minikube.
+Follow the steps below to install the Konveyor Operator in the `my-konveyor-operator` namespace (default) on any Kubernetes distribution, including minikube.
 
 **Procedure**
-1. Install the Konveyor Operator.
+{{< tabs >}}
+{{% tab name="Latest Release" %}}
+1. Install the latest released Konveyor Operator.
 ```
-[user@user ~]$ kubectl apply -f https://raw.githubusercontent.com/konveyor/tackle2-operator/main/tackle-k8s.yaml
+[user@user ~]$ kubectl create -f https://operatorhub.io/install/konveyor-0.1/konveyor-operator.yaml
 ```
-This step will create the konveyor-tackle namespace, catalogsource and other OLM related objects.
+This step will create the `my-konveyor-operator` namespace, catalogsource and other OLM related objects.
+{{% /tab %}}
+{{% tab name="Alpha Release" %}}
+### Installing the alpha version
 
-### Installing _beta_ (or special branches)
+If you need to deploy a latest alpha release build please use the below url
 
-If you need to deploy a beta release (or special branch) please replace the *main* branch in URL with the desired beta branch (i.e v2.0.0-beta.0):
+```
+[user@user ~]$ kubectl create -f https://operatorhub.io/install/alpha/konveyor-operator.yaml
+```
+{{% /tab %}}
+{{% tab name="Development" %}}
+### Installing the latest version
 
-`$ kubectl apply -f https://raw.githubusercontent.com/konveyor/tackle2-operator/<release>-beta.0/tackle-k8s.yaml`
+If you need to deploy a latest available build please follow the steps below,
 
-**Note:** Upgrades between beta releases are **not guaranteed** , once installed, we strongly suggest to edit your subscription and switch to Manual upgrade mode for beta releases: `$ kubectl edit subscription` -> installPlanApproval: Manual
+* Create a custom CatalogSource
+```
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: konveyor
+  namespace: konveyor-tackle
+spec:
+  displayName: Konveyor Operator
+  publisher: Konveyor
+  sourceType: grpc
+  image: quay.io/konveyor/tackle2-operator-index:latest
+```
 
+* Create a Subscription using the custom Catalog Source
+```
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: konveyor-operator
+  namespace: konveyor-tackle
+spec:
+  channel: development
+  installPlanApproval: Automatic
+  name: konveyor-operator
+  source: konveyor
+  sourceNamespace: konveyor-tackle
+```
+* Create the Tackle instance using the url below,
+```
+kubectl create -f https://raw.githubusercontent.com/konveyor/tackle2-operator/main/tackle-k8s.yaml
```

Comment:
Note we just want the Tackle CR created here.

The referenced file has different content we don't want for this step:

https://raw.githubusercontent.com/konveyor/tackle2-operator/main/tackle-k8s.yaml

```
---
apiVersion: v1
kind: Namespace
metadata:
  name: konveyor-tackle
---
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: konveyor
  namespace: konveyor-tackle
spec:
  displayName: Konveyor Operator
  publisher: Konveyor
  sourceType: grpc
  image: quay.io/konveyor/tackle2-operator-index:latest
---
apiVersion: operators.coreos.com/v1
kind: OperatorGroup
metadata:
  name: konveyor
  namespace: konveyor-tackle
spec:
  targetNamespaces:
    - konveyor-tackle
---
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: konveyor-operator
  namespace: konveyor-tackle
spec:
  channel: development
  installPlanApproval: Automatic
  name: konveyor-operator
  source: konveyor
  sourceNamespace: konveyor-tackle
```